backend/global_var.py

```diff
@@ -70,63 +70,66 @@
 ]
 
 # Embassy/consulate attributes
-# Tuple[name_cn, name_en, code, sys, region, continent, country, timezone, ais_code]
+# Tuple[name_cn, name_en, code, sys, region, continent, country, timezone, ais_code, additional_info]
+ADDINFO = {
+    'DOMESTIC': [("test_cn", "test_en"), ("test_cn2", "test_en2")],
+}
```

Comment:
1. `ADDITIONAL_INFO` would be a better name imo.
2. Will it be deterministic that all additional info will be identical among the region instead of country? The region `DOMESTIC` is more of a special case. Maybe we should use country code as the key.
3. Probably don't append it in `EMBASSY_ATTR`? We can alway do a cross-reference or re-construction in the frontend. I propose we add a new field in `/visastatus/meta`'s return dictionary.

Reply:
country level更好一些